README.md

```diff
@@ -38,6 +38,7 @@ An example `.envrc` is provided for optional but recommended use with [`direnv`]
 * `DATABASE_URL` -- A [libpq-compatible postgres uri](https://www.postgresql.org/docs/current/libpq-connect.html#id-1.7.3.8.3.6)
 * `POSTMARK_TOKEN` -- the token from the transactional stream from a [postmark](https://postmarkapp.com) account
 * `EMAIL_ADDRESS` -- the address this deployment should send from
+* `ENCRYPTION_KEYS` -- Comma-joined url-safe-no-pad base64'ed 16 byte cryptographically-random keys. The first one will be used for encryption.
```

Comment:
An "encryption key" which is "used for encryption" is rather broad. Could this be more detailed?
```suggestion
* `DATABASE_ENCRYPTION_KEYS` -- Comma-joined url-safe-no-pad base64'ed 16 byte cryptographically-random keys. The first one will be used to encrypt aggregator API authentication tokens at rest in the database.
```

Reply:
In response to the env var name not the documentation: I was intentionally leaving this generic, just as the aad is not specifically tied to the db. If we have other needs for symmetric encryption in this repo in the future, we can use the same Crypter

Reply:
Re-using a single key for multiple cryptographic applications is generally a bad idea (it tends to invalidate security proofs), so to do what you describe we'd have to extend `Crypter` so that it can use a domain separation tag to derive per-application secrets from a master secret. And then we'd have a chance to revisit this documentation anyway, so I don't think being vague today is meaningfully future-proofing us.